Task_Position_the_Cubes/index.html

```diff
@@ -1,15 +1,23 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Cubes Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
```

Comment:
The 'div' element on line 9 has an attribute 'top' which is not a valid HTML attribute.